src/Test/Utilities/Portable/Roslyn.Test.Utilities.csproj

```diff
@@ -6,8 +6,7 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <OutputType>Library</OutputType>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>netstandard1.3;$(RoslynPortableTargetFrameworks);net46</TargetFrameworks>
-    <DefineConstants>$(DefineConstants);DNX</DefineConstants>
+    <TargetFrameworks>$(RoslynPortableTargetFrameworks)</TargetFrameworks>
```

Comment:
I think this is the wrong approach. This DLL deliberately targets netstandard1.3 in order to avoid multi-targeting all the places you just added. The design of this DLL is to provide a netstandard1.3 API set with a net46 and netcoreapp2.1 implementation.